### ISSUE TITLE
Suggestion about my animelist part of extension 

### ISSUE THREAD

**Issue by KingFishy492** [lap00zza/9anime-Companion#99]
- [ ] Bug
- [ ] Question
- [x] Suggestion
- [ ] Other (please describe):

**Describe your issue:**
I see that when you start an anime well add it to your list using the app on 9anime it adds a start date is it possible to when you reac the end of the anime to be able to have it add to then add it at the end of the show when you click the last time to add it to 12-12 and it adds the date to myanimelist so you dont have to go and add it yourself or work out when you finished watching it?

**Comment by lap00zza** [comment]
Sure thing :) its actually a planned feature and a bit similar to #68

**Comment by KingFishy492** [comment]
Yes i see it is witch is awesome with what they have said that would be useful with not having to go back and forth between sites, back to the question above i asked well suggestion more or less will this do it automaticlly to all anime when its added the end time even if you had already added it to your watching list or planned to watch list so you could be going threw and go oh thats cool and add it on my anime and it would then still add the start and end date? so from the first time you hit the button it marks the start date for 1+ and when you get to the end it would put the end date at 13+ or how ever long the show run is
